oshi-core/src/main/java/oshi/hardware/GlobalMemory.java

```diff
@@ -36,14 +36,14 @@
      *
      * @return Total number of bytes.
      */
-    long getTotal();
+    long getMemTotal();

```

Comment:
We can't change the API without a major version bump.  If you want to change the getter for readability you need to keep the old name around (deprecated) until the next major version bump.